mlir/utils/jenkins/Jenkinsfile

```diff
@@ -6,7 +6,9 @@ void buildProject(String target, String cmakeOpts) {
             buildType: 'RelWithDebInfo',\
             installation: 'InSearchPath',\
             steps: [[args: target]],\
-            cmakeArgs: "$cmakeOpts"
+            cmakeArgs: """-DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
```

Comment:
Just a curiosity: should this be changed in `Jenkinsfile.downstream` and `jenkinsfile.release` as well?